sierra_adapter/sierra_item_merger/src/main/scala/uk/ac/wellcome/platform/sierra_item_merger/modules/SierraItemMergerModule.scala

```diff
@@ -0,0 +1,29 @@
+package uk.ac.wellcome.platform.sierra_item_merger.modules
+
+import akka.actor.ActorSystem
+import com.twitter.inject.{Injector, TwitterModule}
+import uk.ac.wellcome.platform.sierra_item_merger.services.SierraItemMergerWorkerService
+import uk.ac.wellcome.utils.TryBackoff
+
+object SierraItemMergerModule extends TwitterModule with TryBackoff {
+
+  override lazy val continuous: Boolean = false
```

Comment:
I can’t remember where @alicefuzier and I got to on this – is non-continuous definitely the right behaviour?

Reply:
No, it's the wrong behaviour. Copy paste error...